### ISSUE TITLE
Use case

### ISSUE THREAD

**Issue by virgesmith** [Robinlovelace/ukboundaries#3]
I think something like this would be really useful:
```
coverage=c("E09000001") # City of London (LAD)
boundary_type="LSOA11"
resolution="Generalised_Clipped"
spdf = getboundaries(coverage, boundary_type, detail)
# returns a spatial data frame containing the LSOAs in City of London
```
I'd be happy to implement this but I'd want it to work with UKCensusAPI and that means writing it in python and exposing to R via reticulate... @Robinlovelace what do you think?

**Comment by Robinlovelace** [comment]
Yes that would be awesome. There is certainly demand for it - just a question of getting support, unless we just go for it. Build it... as Alistair Rae says here: https://twitter.com/undertheraedar/status/976423074131271680

**Comment by Robinlovelace** [comment]
Heads-up @virgesmith this is worth looking at - popular and much-loved example of how a package like this can be useful to 100s of people: https://github.com/walkerke/tidycensus
